.github/workflows/flaky-test-monitor.yml

```diff
@@ -5,10 +5,10 @@ name: Flaky Test Monitor
 on:
   schedule:
     - cron: '0 */2 * * *' # every 2 hours
-  push:
-    paths:
-      - '.github/workflows/flaky-test-monitor.yml'
-      - '.github/workflows/ci.yml'
+#  push:
+#    paths:
+#      - '.github/workflows/flaky-test-monitor.yml'
+#      - '.github/workflows/ci.yml'
```

Comment:
Should this be un-commented or removed?

Reply:
Thanks. Flaky Test Monitor CI workflow needs to have a major overhaul now that the main CI workflow uses custom runners and Docker caching. I will address that in a separate PR.

6761410928469ed7235afe5708a852ebf76871e6